src/Compilers/VisualBasic/Portable/Errors/Errors.vb

```diff
@@ -2015,7 +2018,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         IDS_LangVersions = 56010
         IDS_ToolName = 56011
 
-        ERR_StdInOptionProvidedButConsoleInputIsNotRedirected = 56032
+        HDN_NextAvailable = 56012
```

Comment:
>HDN_NextAvailable = 56012 [](http://example.com/codeflow?start=8&length=25)

It is not clear why leave previous INF/IDS values in the range of hidden diagnostics. I am not even sure if we have to maintain specific values for them.